webhook_to_fedora_messaging/endpoints/user.py

```diff
@@ -0,0 +1,58 @@
+from flask import Blueprint, Flask, request, Response, Request
+from ..database import db
+from ..models.user import User
+from sqlalchemy_helpers import get_or_create
+
+app = Flask(__name__)
+user_endpoint = Blueprint("user_endpoint", __name__)
+
+@user_endpoint.route("/user", methods=["POST"])
+def create_user():
+    """
+        Used for creating a new user by sending a post request to /user/ path.
```

Comment:
Get rid of the added tab please.

```suggestion
    Used for creating a new user by sending a post request to /user/ path.
```